Makefile

```diff
@@ -321,6 +337,19 @@ az-patch-install-helm: ## Update Azure client env vars and settings in helm valu
 
 	helm install kaito-workspace ./charts/kaito/workspace --namespace $(KAITO_NAMESPACE) --create-namespace
 
+.PHONY: az-patch-install-ragengine-helm
+az-patch-install-ragengine-helm:
+	az aks get-credentials --name $(AZURE_CLUSTER_NAME) --resource-group $(AZURE_RESOURCE_GROUP)
+
+	yq -i '(.image.repository)                                              = "$(REGISTRY)/ragengine"'                    ./charts/kaito/ragengine/values.yaml
+	yq -i '(.image.tag)                                                     = "$(IMG_TAG)"'                               ./charts/kaito/ragengine/values.yaml
+	if [ $(TEST_SUITE) = "azkarpenter" ]; then \
```

Comment:
We do you need to change the feature gate here? This feature gate is removed by Linbo's change.